engine/engine/src/engine.cpp

```diff
@@ -590,6 +634,12 @@ namespace dmEngine
 #endif
         }
 
+        const char* alt_ssl_key_path = dmConfigFile::GetString(engine->m_Config, "network.ssl_certificates", "");
+        if (LoadAndSetSslKeys(res_path, content_root, alt_ssl_key_path))
```

Comment:
Since it's supposed to try each path, I don't suggest us adding more arguments for each path we add.
Instead, either:
1) make the function return ok/fail and call it in a loop
```
const char* paths[] = {archive_path, project_root, alt_ssl_key_path};
for (uint32_t i = 0; i < DM_ARRAY_SIZE(paths); ++i) {
    if (LoadAndSetSslKeys(paths[i], SSL_KEYS_FILENAME)) {
        break;
    }
}
```

2) or pass the list of paths into the function
```
const char* paths[] = {archive_path, project_root, alt_ssl_key_path};
LoadAndSetSslKeys(SSL_KEYS_FILENAME, paths, DM_ARRAY_SIZE(paths));
```